workshop/README.md

```diff
@@ -270,79 +364,93 @@ The idea for this transaction is that it will accept only one value that represe
 }
 ```
 
-**Note:** For each step, the solution can be found in the section below called `Solution: Payment Transaction`.
+**Note:** For each step, the solution can be found in the section below called `Solution: Payment Transaction`.<br><br>
 
 #### Steps To Be Implemented
-1. Replace the `super` cache call with the [actual implementation in TransferTransaction](https://github.com/LiskHQ/lisk-sdk/blob/development/elements/lisk-transactions/src/0_transfer_transaction.ts#L69). Remember the task about reading up on combining AND and OR, this is exactly what happens here.<br><br>
+1. Replace the `super` cache call with the [actual implementation in the TransferTransaction class](https://github.com/LiskHQ/lisk-sdk/blob/development/elements/lisk-transactions/src/0_transfer_transaction.ts#L69). 
```

Comment:
Why is this needed?

Reply:
In case the logic for transfertransaction changes, it might cache something else. But should be better removed in this case.